www/include/options/accessLists/resourcesACL/listsResourcesAccess.php

```diff
@@ -38,18 +38,26 @@
 }
 
 include("./include/common/autoNumLimit.php");
+$num = (int) $num;
+$limit = (int) $limit;
```

Comment:
same

Reply:
all the checks are already done in autoNumLimit.php